main.go

```diff
@@ -164,14 +184,14 @@ func main() {
 		go runServer(ctx, &serverWg, asyncServer, serverErrCh)
 
 		if config.EnableHTTP {
-			httpServer := server.NewHTTPServer(shardManager)
+			httpServer := server.NewHTTPServer(shardManager, wl)
```

Comment:
With all the complexity that will be involved in capturing commands in WAL, should we now run Server with one protocol at a given time?